hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreScanner.java

```diff
@@ -205,6 +207,9 @@ private StoreScanner(HStore store, Scan scan, ScanInfo scanInfo,
       this.scanUsePread = this.readType != Scan.ReadType.STREAM;
     }
     this.preadMaxBytes = scanInfo.getPreadMaxBytes();
+    // TODO : Introduce config here at the ScanInfo level. Determine based on number of blocks
```

Comment:
This would be something deeply internal that no user could meaningfully set unless they read all the code and become a wizard. Please figure this out based on current configuration. 

No new config!